lib/GeoIP2/WebService/Client.pm

```diff
@@ -378,17 +390,17 @@ This method creates a new client object. It accepts the following arguments:
 
 =over 4
 
-=item * user_id
+=item * account_id
 
-Your MaxMind User ID. Go to L<https://www.maxmind.com/en/my_license_key> to see
-your MaxMind User ID and license key.
+Your MaxMind Account ID. Go to L<https://www.maxmind.com/en/my_license_key> to see
+your MaxMind Account ID and license key.
```

Comment:
Not a change request, but if we're leaving some type of backwards compatibility with regards to user_id/account_id, it make by helpful to acknowledge that here.